packages/ast-to-zql/src/bin.ts

```diff
@@ -54,7 +54,11 @@ function readFromStdin(): Promise<string> {
 
 async function formatOutput(content: string): Promise<string> {
   try {
-    const options = (await resolveConfig(new URL(import.meta.url))) ?? {};
+    // Resolve prettier config from current working directory. If the config
+    // file is in the cwd we add one more path component to the path to make
```

Comment:
the comment says `_if_ the config file` but the code unconditionally adds a path component.